conf/upgrade/upgrade_3m_3w.yaml

```diff
@@ -3,4 +3,5 @@ REPORTING:
   polarion:
     upgrade_id: 'OCS-1484'
 UPGRADE:
+  skip_upgrade: true
```

Comment:
How and where is this skip_upgrade used?

As this one is gneric file we use in our production runs, I probably don't want to have this changed here.

The main purpose of this file is for have proper polarion ID from production jobs. So please remove it from here.